### DISCUSSION TITLE
Raspberry Pi Theme  3.5" Landscape

### DISCUSSION THREAD

**Discussion by MarkOliver1959** [mathoudebine/turing-smart-screen-python#353] [Themes]
First theme that is to aid people that use Raspberry Pi's in headless mode and need to check if their devices are working and connected to their network etc. 

This theme makes use of the new Custom Data Sources in Release 3.3.3 to obtain some additional O/S and IP info. 

A bit of caution if you have already modified the sensors_custom.py as unzipping this file within the repo directory would overwrite it.

[PiSystem.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12818830/PiSystem.zip)
![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/44792882/c8c21879-2a28-4221-a908-f2f024c6b11a)

**Comment by mathoudebine** [comment]
Amazing work ! Thanks for sharing ❤️

**Reply by MarkOliver1959** [reply]
Thank you Matthiew.

If anybody wants - I can create a Portrait version and also make it more Raspberry themed with fully Raspberry Pi palette ?

I'm also looking into using ESP32 touch displays and making them interwork with your project. The purpose is to provide a method of rebooting or some configuration similar to the screens below, which is something I wrote for the Raspberry a long time ago. Although it would be way better if the company that makes the Turing Smart, produced a touch version - hint hint :-) 
  
![Main](https://github.com/mathoudebine/turing-smart-screen-python/assets/44792882/9b8e8987-a28d-4e2c-982f-364486a35ce5)
![SysInfo](https://github.com/mathoudebine/turing-smart-screen-python/assets/44792882/c537ff0c-e6f2-4814-8489-0fce6e4a5892)

**Reply by mathoudebine** [reply]
I agree with you, I'd like a Turing touch screen! If the Turing company sees this ... 😉
